
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


cat: write error: Broken pipe
2023-05-01 16:21:33.763118
2023-05-01 16:21:34.249861
2023-05-01 16:21:34.664887

real	0m1.946s
user	0m1.636s
sys	0m0.226s
cat: write error: Broken pipe
2023-05-01 16:21:45.889746
2023-05-01 16:21:46.265849
2023-05-01 16:21:46.670554

real	0m1.695s
user	0m1.486s
sys	0m0.179s
cat: write error: Broken pipe
2023-05-01 16:21:57.851913
2023-05-01 16:21:58.164347
2023-05-01 16:21:58.559798

real	0m1.593s
user	0m1.397s
sys	0m0.173s
cat: write error: Broken pipe
2023-05-01 16:22:09.757242
2023-05-01 16:22:10.122947
2023-05-01 16:22:10.524757

real	0m1.668s
user	0m1.493s
sys	0m0.153s
cat: write error: Broken pipe
2023-05-01 16:22:21.871363
2023-05-01 16:22:22.772990
2023-05-01 16:22:23.262422

real	0m2.445s
user	0m2.230s
sys	0m0.193s
cat: write error: Broken pipe
2023-05-01 16:22:34.588097
2023-05-01 16:22:35.479401
2023-05-01 16:22:36.386059

real	0m2.848s
user	0m2.657s
sys	0m0.166s
cat: write error: Broken pipe
2023-05-01 16:22:47.614893
2023-05-01 16:22:48.566835
2023-05-01 16:22:50.014602

real	0m3.345s
user	0m3.031s
sys	0m0.282s
cat: write error: Broken pipe
2023-05-01 16:23:01.172332
2023-05-01 16:23:01.371073
2023-05-01 16:23:01.496379

real	0m1.189s
user	0m1.036s
sys	0m0.132s
mkdir: cannot create directory ‘SCODE_D_2’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_4’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_6’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_8’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
/data/gpfs/projects/punim0638/stephenz/miniconda3/lib/python3.10/site-packages/Scribe-0.0.0-py3.10.egg/Scribe/causal_network.py:18: UserWarning:  WARNING: No expression data argument given. if you intend to load the data from a file, call the method 'read_expression_file'.
  if expression==None: warnings.warn(" WARNING: No expression data argument given. if you intend to load the data from a file, call the method 'read_expression_file'.")
/data/gpfs/projects/punim0638/stephenz/miniconda3/lib/python3.10/site-packages/Scribe-0.0.0-py3.10.egg/Scribe/causal_network.py:298: FutureWarning: Could not cast to int64, falling back to object. This behavior is deprecated. In a future version, when a dtype is passed to 'DataFrame', either all columns will be cast to that dtype, or a TypeError will be raised
  max_rdi_delay = pd.DataFrame({node_id: [np.nan for i in self.node_ids] for node_id in self.node_ids}, index=self.node_ids,
Create causal_model successfully
  Activating project at `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-LI/dyn-LI-1000-6/pidc_env`
    Updating registry at `~/.julia/registries/General.toml`
   Resolving package versions...
  No Changes to `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-LI/dyn-LI-1000-6/pidc_env/Project.toml`
  No Changes to `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-LI/dyn-LI-1000-6/pidc_env/Manifest.toml`
Getting nodes...
Inferring network...
